pkg/ingester/ingester.go

```diff
@@ -1400,7 +1413,7 @@ func (i *Ingester) Push(ctx context.Context, req *cortexpb.WriteRequest) (*corte
 		if errors.As(firstPartialErr, &ve) {
 			code = ve.code
 		}
-		level.Debug(logutil.WithContext(ctx, i.logger)).Log("msg", "partial failures to push", "totalSamples", succeededSamplesCount+failedSamplesCount, "failedSamples", failedSamplesCount, "firstPartialErr", firstPartialErr)
+		level.Debug(logutil.WithContext(ctx, i.logger)).Log("msg", "partial failures to push", "totalSamples", succeededSamplesCount+succeededHistogramCount+failedSamplesCount+failedHistogramCount, "failedSamples", failedSamplesCount, "failedHistogram", failedHistogramCount, "firstPartialErr", firstPartialErr)
```

Comment:
If we want to separate out samples and histograms then let's add a dedicated `totalHistograms`.

failedHistogram => failedHistograms